Buttplug.Server.Managers.ErosTek/ET312Protocol.cs

```diff
@@ -0,0 +1,579 @@
+        /// </summary>
+        public enum EEPROM : uint
+        {
+            BaseEEPROM = 0x8000,                // unused
+            IsProvisioned = 0x8001,             // Magic (0x55 means we’re provisioned)
+            BoxSerialLow = 0x8002,              // Box Serial 1
+            BoxSerialHigh = 0x8003,             // Box Serial 2
+            Adr8004 = 0x8004,                   // unused, set to 0x00
+            Adr8005 = 0x8005,                   // unused, set to 0x00
+            ELinkSig1 = 0x8006,                 // ELinkSig1 - ELINK_SIG1_ADDR (default 0x01)
+            ELinkSig2 = 0x8007,                 // ELinkSig2 - ELINK_SIG2_ADDR (default 0x01)
+            TopModeNV = 0x8008,                 // TopMode NonVolatile (written during routine write)
+            PowerLevelNV = 0x8009,              // Power Level
+            SplitModeANV = 0x800A,              // Split A Mode Num
+            SplitModeBNV = 0x800B,              // Split B Mode Num
+            FavouriteModeNV = 0x800C,           // Favourite Mode
+            AdvancedRampLevelNV = 0x800D,       // Advanced Parameter: RampLevel
+            AdvancedRampTimeNV = 0x800E,        // Advanced Parameter: RampTime
+            AdvancedDepthNV = 0x800F,           // Advanced Parameter: Depth
+            AdvancedTempoNV = 0x8010,           // Advanced Parameter: Tempo
+            AdvancedFrequencyNV = 0x8011,       // Advanced Parameter: Frequency
+            AdvancedEffectNV = 0x8012,          // Advanced Parameter: Effect
+            AdvancedWidthNV = 0x8013,           // Advanced Parameter: Width
+            AdvancedPaceNV = 0x8014,            // Advanced Parameter: Pace
+            Adr8015 = 0x8015,                   // set to 0x00"
+            Adr8016 = 0x8016,                   // set to 0x00"
+            Adr8017 = 0x8017,                   // set to 0x00"
+            UserRoutineVector1 = 0x8018,        // Start Vector User 1 - COMM_USER_BASE
+            UserRoutineVector2 = 0x8019,        // Start Vector User 2
+            UserRoutineVector3 = 0x801A,        // Start Vector User 3
+            UserRoutineVector4 = 0x801B,        // Start Vector User 4
+            UserRoutineVector5 = 0x801C,        // Start Vector User 5
+            UserRoutineVector6 = 0x801D,        // Start Vector User 6
+            UserRoutineVector7 = 0x801E,        // Start Vector User 7 (not implemented)
+            UserRoutineVector8 = 0x801F,        // Start Vector User 8 (not implemented)
+            UserRoutinePointersA = 0x8020,      // User routine module pointers 0x80-0x9f
+            UserSpaceA = 0x8040,                // Space for User Modules
+            UserRoutinePointersB = 0x8100,      // User routine module pointers 0xa0-0xbf
+            UserSpaceB = 0x8120,                // Space for User Modules
+        }
+
+        // Possible bit field values for SystemFlags
+        [Flags]
+        public enum SystemFlags : byte
```

Comment:
You started to use `///` (tripple-slash) comments above, don't forget about the rest 😜

Reply:
Sure thing. 

By the way, when are you going to start reviewing the entire existing code by the same standards? 

:)

Reply:
It's easier to focus on PRs to keep up the code quality. As I work on code myself, i'll make changes as i see them.